observability/cockpit/how-to/send-metrics-logs-to-cockpit.mdx

```diff
@@ -95,7 +95,7 @@ You can push logs with any Loki compatible agent such as the [Promtail](https://
   ### Creating a Grafana dashboard and visualizing your Python logs
 
     1. [Log into Grafana](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/) using your [Grafana credentials](/observability/cockpit/how-to/retrieve-grafana-credentials/).
-    2. Click the **Home** icon, then click **Dashboards**.
+    2. Click the **Toggle menu** icon, then click **Dashboards**.
```

Comment:
If it is an icon and not text that says "Toggle menu" I think it shouldn't be in bold